envs/monkey_zoo/blackbox/island_client/monkey_island_client.py

```diff
@@ -113,6 +122,7 @@ def reset_island(self):
         self._reset_simulation_data()
         self._reset_credentials()
         self._reset_island_mode()
+        self.set_masque(b"")
```

Comment:
I think when the endpoint to reset the island mode is called, this should happen automatically. I'm working on a PR for that.

Reply:
This is now unnecessary due to PR #3257. That being said, I don't mind leaving it. It's explicit.